[Snyk] Upgrade cropperjs from 1.5.12 to 1.6.2

alikon opened the pull request:
<p>This PR was automatically created by Snyk using the credentials of a real user.</p><br /><h3>Snyk has created this PR to upgrade cropperjs from 1.5.12 to 1.6.2.</h3>

:information_source: Keep your dependencies up-to-date. This makes it easier to fix existing vulnerabilities and to more quickly identify and fix newly disclosed vulnerabilities when they affect your project.
<hr/>

- The recommended version is **4 versions** ahead of your current version.
- The recommended version was released **22 days ago**, on 2024-04-21.


<details>
<summary><b>Release notes</b></summary>
<br/>
  <details>
    <summary>Package name: <b>cropperjs</b></summary>
    <ul>
      <li>
        <b>1.6.2</b> - <a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/releases/tag/v1.6.2">2024-04-21</a></br><ul>
<li>Disable touch callout on iOS Webkit-based browsers (<a class="issue-link js-issue-link" data-error-text="Failed to load title" data-id="2207186579" data-permission-text="Title is private" data-url="https://github.com/fengyuanchen/cropperjs/issues/1155" data-hovercard-type="pull_request" data-hovercard-url="/fengyuanchen/cropperjs/pull/1155/hovercard" href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/pull/1155">#1155</a>).</li>
</ul>
      </li>
      <li>
        <b>1.6.1</b> - <a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/releases/tag/v1.6.1">2023-09-17</a></br><ul>
<li>Fix an issue where some CSS styles were incompatible with old browsers.</li>
</ul>
      </li>
      <li>
        <b>1.6.0</b> - <a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/releases/tag/v1.6.0">2023-08-26</a></br><ul>
<li>Add a <code>rounded</code> option to the <code>getCroppedCanvas</code> method (<a class="issue-link js-issue-link" data-error-text="Failed to load title" data-id="1604450187" data-permission-text="Title is private" data-url="https://github.com/fengyuanchen/cropperjs/issues/1047" data-hovercard-type="pull_request" data-hovercard-url="/fengyuanchen/cropperjs/pull/1047/hovercard" href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/pull/1047">#1047</a>).</li>
</ul>
      </li>
      <li>
        <b>1.5.13</b> - <a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/releases/tag/v1.5.13">2022-11-20</a></br><ul>
<li>Add <code>backface-visibility: hidden</code> to the cropper image to avoid ghost lines when moving (<a class="issue-link js-issue-link" data-error-text="Failed to load title" data-id="4389723" data-permission-text="Title is private" data-url="https://github.com/fengyuanchen/cropperjs/discussions/982" data-hovercard-type="discussion" data-hovercard-url="/fengyuanchen/cropperjs/discussions/982/hovercard" href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/discussions/982">#982</a>).</li>
<li>Remove useless <code>cropper-hide</code> class from cropper image.</li>
<li>Check whether the parent node exists or not before removing the cropper container.</li>
</ul>
      </li>
      <li>
        <b>1.5.12</b> - 2021-06-12
      </li>
    </ul>
    from <a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/releases">cropperjs GitHub release notes</a>
  </details>
</details>


<details>
  <summary><b>Commit messages</b></summary>
  </br>
  <details>
    <summary>Package name: <b>cropperjs</b></summary>
    <ul>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/da3f6f7d4b9c29c7234ddf114c71a27a269855f0">da3f6f7</a> build: release 1.6.2</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/388368d7184c53958ecfd151f1bfc98436b33dc7">388368d</a> build: update some dependencies</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/216178212338eda7cd8b0a57feee718b1dfed720">2161782</a> build: downgrade cpy-cli to v3.x</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/24dbf99d0fcd7109d24663e358249b450feac300">24dbf99</a> fix: disable touch callout on iOS webkit-based browsers</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/a444f6fa051844dbf22434d597d023f570974da5">a444f6f</a> fix: disable long press context menu on iOS chorme (#1155)</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/b122bb6769e867e867ee6a913e12231cbcdf5463">b122bb6</a> build: release 1.6.1</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/dda8f95b978f274bcff16ccacfcec35092172caa">dda8f95</a> build: downgrade stylelint to 13 to avoid side effects</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/72bb6d84f84593386f68724cf80c04fd8e2c3f01">72bb6d8</a> docs: update comment description</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/92582ee145754fbe815718b7a9ca0c59c13d0264">92582ee</a> build: release 1.6.0</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/f652134403f33ea89415fb565653064409445b16">f652134</a> docs: update Related projects add Symfony UX Cropper.js (#1097)</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/3957a64bb0e41e7bec96bd7b0030a174ee8288f6">3957a64</a> refactor: improve code and add changelog</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/4310a8e33f75a0052dd0903dbd141e602add6435">4310a8e</a> feat: add &#x60;rounded&#x60; option to &#x60;getCroppedCanvas&#x60; method (#551) (#1047)</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/759d65e94d9c120684f2fabcdf8f2455041acfc8">759d65e</a> docs: sort projects</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/1e76d4169068c1566f212f654f1fe963608d76fe">1e76d41</a> docs: improve example for dynamic aspect ratio</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/27e4f593028f7a6403f91387f99eb08b208103f8">27e4f59</a> docs: fix missing backtick (#1012)</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/336bc89b275aa73b8a4d9ef7973f86118ed1c56b">336bc89</a> docs: fix icon font url</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/3375ce08e1fc66cf61392dc082873ed1c9e13333">3375ce0</a> build: release 1.5.13</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/97d2c2fd8b9a04dd21e09dbfa296ce8c5e806585">97d2c2f</a> fix: add &#x60;backface-visibility: hidden&#x60; to cropper image to avoid ghost lines when moving</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/3176f04b5ea4a41bb2f6aa74c2383cd998c72556">3176f04</a> fix: remove useless &#x60;cropper-hide&#x60; class from cropper image</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/0c75b668e8a12855044f9bf8c95e9f85e1682a4b">0c75b66</a> docs: add blazor-cropper repo url; add reference to profile users GitHub (#997)</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/dfba39d7752222f0e9380d9eb3eb432eea2d96bd">dfba39d</a> docs: improve demo code for &#x60;getCroppedCanvas&#x60; method</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/f1bd5ab5232317572cb43781ff91274dcebd7e22">f1bd5ab</a> docs: add comment to hack code for Safari</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/cee3b0e503f3c407fabe0122b84a98b83bba89f7">cee3b0e</a> fix: example for round image on iOS/Safari (#978)</li>
      <li><a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/commit/133b8b07d00a68a01063af2f57f0fab183050fcf">133b8b0</a> fix: check whether the parent node exists or not before removing the cropper container</li>
    </ul>

   <a href="https://snyk.io/redirect/github/fengyuanchen/cropperjs/compare/803f928cf0f41b409e39c4666bc49d93ca98f791...da3f6f7d4b9c29c7234ddf114c71a27a269855f0">Compare</a>
  </details>
</details>
<hr/>

**Note:** *You are seeing this because you or someone else with access to this repository has authorized Snyk to open upgrade PRs.*

For more information:  <img src="https://api.segment.io/v1/pixel/track?data=eyJ3cml0ZUtleSI6InJyWmxZcEdHY2RyTHZsb0lYd0dUcVg4WkFRTnNCOUEwIiwiYW5vbnltb3VzSWQiOiI5NzhmNjg3OC04YmE5LTQ5YWMtYjM5Zi0yMGY0MDNkMGMyMDEiLCJldmVudCI6IlBSIHZpZXdlZCIsInByb3BlcnRpZXMiOnsicHJJZCI6Ijk3OGY2ODc4LThiYTktNDlhYy1iMzlmLTIwZjQwM2QwYzIwMSJ9fQ==" width="0" height="0"/>

🧐 [View latest project report](https://app.snyk.io/org/alikon/project/7f972e16-a9b0-41f9-a764-0801652e912f?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;upgrade-pr)

🛠 [Adjust upgrade PR settings](https://app.snyk.io/org/alikon/project/7f972e16-a9b0-41f9-a764-0801652e912f/settings/integration?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;upgrade-pr)

🔕 [Ignore this dependency or unsubscribe from future upgrade PRs](https://app.snyk.io/org/alikon/project/7f972e16-a9b0-41f9-a764-0801652e912f/settings/integration?pkg&#x3D;cropperjs&amp;utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;upgrade-pr#auto-dep-upgrades)

<!--- (snyk:metadata:{"prId":"978f6878-8ba9-49ac-b39f-20f403d0c201","prPublicId":"978f6878-8ba9-49ac-b39f-20f403d0c201","dependencies":[{"name":"cropperjs","from":"1.5.12","to":"1.6.2"}],"packageManager":"npm","type":"auto","projectUrl":"https://app.snyk.io/org/alikon/project/7f972e16-a9b0-41f9-a764-0801652e912f?utm_source=github&utm_medium=referral&page=upgrade-pr","projectPublicId":"7f972e16-a9b0-41f9-a764-0801652e912f","env":"prod","prType":"upgrade","vulns":[],"issuesToFix":[],"upgrade":[],"upgradeInfo":{"versionsDiff":4,"publishedDate":"2024-04-21T07:55:24.150Z"},"templateVariants":[],"hasFixes":false,"isMajorUpgrade":false,"isBreakingChange":false,"priorityScoreList":[]}) --->

alikon commented:
i hate some  CS rules

richard67 commented:
Yes, you are Italian, so you hate rules, like e.g. traffic rules ;-) (teasing joke, don't take it serious)

snyk-bot committed:
fix: upgrade cropperjs from 1.5.12 to 1.6.2

Snyk has created this PR to upgrade cropperjs from 1.5.12 to 1.6.2.

See this package in npm:
https://www.npmjs.com/package/cropperjs

See this project in Snyk:
https://app.snyk.io/org/alikon/project/7f972e16-a9b0-41f9-a764-0801652e912f?utm_source=github&utm_medium=referral&page=upgrade-pr
